rootfs/usr/share/inputplumber/capability_maps/zone_type1.yaml

```
@@ -0,0 +1,145 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/ShadowBlip/InputPlumber/main/rootfs/usr/share/inputplumber/schema/capability_map_v2.json
+# Schema version number
+version: 2
+
+# The type of configuration schema
+kind: CapabilityMap
+
+# Name for the device event map
+name: Zotac Zone Type 1
+
+# Unique identifier of the capability mapping
+id: zone1
+
+# List of mapped events that are activated by a specific set of activation keys.
+mapping:
+  - name: Screen Left (Short)
+    source_events:
+      - evdev:
+          event_type: KEY
+          event_code: KEY_F16
+          value_type: button
+    target_event:
+      gamepad:
+        button: Guide
+  - name: Screen Left (Long)
+    source_events:
+      - evdev:
+          event_type: KEY
+          event_code: KEY_F18
+          value_type: button
+    target_event:
+      gamepad:
+        button: Keyboard
+  - name: Screen Right 1
+    source_events:
+      - evdev:
+          event_type: KEY
+          event_code: KEY_F17
+          value_type: button
+    target_event:
+      gamepad:
+        button: QuickAccess
+  - name: Screen Right 2
+    source_events:
+      - evdev:
+          event_type: KEY
+          event_code: KEY_F19
+          value_type: button
+    target_event:
+      gamepad:
+        button: QuickAccess2
+  - name: Left Paddle
+    source_events:
+      - evdev:
+          event_type: KEY
+          event_code: KEY_F14
+          value_type: button
+    target_event:
+      gamepad:
+        button: LeftPaddle1
+  - name: Right Paddle
+    source_events:
+      - evdev:
+          event_type: KEY
+          event_code: KEY_F15
+          value_type: button
+    target_event:
+      gamepad:
+        button: RightPaddle1
+  - name: Left Dial Counter-clockwise
+    source_events:
+      - evdev:
+          event_type: REL
+          event_code: REL_HWHEEL
+          axis_direction: negative
+          value_type: button
+    target_event:
+      keyboard: KeyVolumeDown
+  - name: Left Dial Clockwise
+    source_events:
+      - evdev:
+          event_type: REL
+          event_code: REL_HWHEEL
+          axis_direction: positive
+          value_type: button
+    target_event:
+      keyboard: KeyVolumeDown
+  - name: Right Dial Counter-clockwise
+    source_events:
+      - evdev:
+          event_type: REL
+          event_code: REL_WHEEL
+          axis_direction: negative
+          value_type: button
+    target_event:
+      keyboard: KeyVolumeDown
+  - name: Right Dial Clockwise
+    source_events:
+      - evdev:
+          event_type: REL
+          event_code: REL_WHEEL
+          axis_direction: positive
+          value_type: button
+    target_event:
+      keyboard: KeyVolumeDown
```

The capability map is meant to be used to fix devices that emit the wrong input events, and map them to the _real_ capabilities of the input device. The Zotac Zone dials are a new type of input, not volume buttons, so these should be mapped to a new "Dial" input capability instead:

```suggestion
      gamepad:
        dial: RightStickDial
```

To do this, you'll need to add the new capability in `capability.rs` and in the config module.

The reason we want to create a new capability for this new type of input is it allows us to distinguish between volume buttons and these dials, so they can be mapped properly in an input profile. Without a new capability, we would not be able to make that distinction.

---

Ah, I see now. I'll add this.